configuration/api_test.go

```diff
@@ -271,12 +274,22 @@ func createMockedResponder(t *testing.T, method string) roundTripFunc {
 			}
 		}
 
-		if req.URL.String() != "https://api.livechatinc.com/v3.4/configuration/action/"+method {
+		expectedMethod := http.MethodPost
```

Comment:
`expectedHttp` por favor, let's not use the same name (method) for two different things

Reply:
Renamed to `expectedHTTPMethod`.